### PR TITLE
Adds a new lobby sound.

### PR DESCRIPTION
**Pull request by Iamgoofball**
:cl: Iamgoofball
rscadd: Added a new lobby menu sound.
/:cl:

It's a surprise! I'm gonna go make a sandwich now. Thanks a lot, bye-bye.

### CODE REVIEW

**Comment by PKPenguin321** [comment]
[i_did_not_grief_them.ogg](http://puu.sh/s1qJc/02ed6c6ce6.mp3)

File: code/world.dm [hunk 1]
```diff
@@ -246,7 +246,7 @@ var/last_irc_status = 0
 		if(ticker && ticker.round_end_sound)
 			world << sound(ticker.round_end_sound)
 		else
-			world << sound(pick('sound/AI/newroundsexy.ogg','sound/misc/apcdestroyed.ogg','sound/misc/bangindonk.ogg','sound/misc/leavingtg.ogg', 'sound/misc/its_only_game.ogg')) // random end sounds!! - LastyBatsy
+			world << sound(pick('sound/AI/newroundsexy.ogg','sound/misc/apcdestroyed.ogg','sound/misc/bangindonk.ogg','sound/misc/leavingtg.ogg', 'sound/misc/its_only_game.ogg', 'sound/misc/i_did_not_grief_them.ogg')) // random end sounds!! - LastyBatsy
```
Comment (JJRcop) [inline code/world.dm]: Wrong list.
[/code/controllers/subsystem/ticker.dm:61](https://github.com/tgstation/tgstation/blob/aac8757b644414514fc1053d758d2fbc587d05a8/code/controllers/subsystem/ticker.dm#L61)

**Review by JJRcop — CHANGES_REQUESTED** [review]
You added it to the round end sounds instead of the lobby sounds. I do like the music, though.

File: code/world.dm (hunk 1 above)
Comment (Iamgoofball) [inline code/world.dm]: sdfasdfsdfjasdfhsdfa

**Comment by Bawhoppen** [comment]
Make a changelog and use the rare sound tag ❕ ❕ ❕ ❕ ❕ ❕ ❕ ❕ ❕

**Comment by lordpidey** [comment]
I've heard this before, but can't remember where for the life of me.

**Comment by Dorsisdwarf** [comment]
eh

**Comment by PKPenguin321** [comment]
not super fond of this myself, too ambient

**Comment by ExcessiveUseOfCobblestone** [comment]
> Not muting the lobby sound so you can hear the midi of the round

**Comment by optimumtact** [comment]
Copyright status?

**Comment by Iamgoofball** [comment]
@optimumtact The record company listed as the label (TPW Records) doesn't seem to actually exist. I've got no fucking clue.

**Pull request merged**